### ISSUE TITLE
New option "tcp-upstream-without-downstream" possible?

### ISSUE THREAD

**Issue by iz8mbw** [NLnetLabs/unbound#230]
Hi!
I see in the documentation the recent option `udp-upstream-without-downstream`.
Well, if need to do the opposite how can I do?
I need to can do only TCP to upstream (since for upstream I use only "DNS over TLS" and DoT supports only TCP for what I know) and let unbound listen only in UDP port 53 without listening also on TCP.
Currently my unbound listen on UDP port 53 but it listen also on TCP port 53 and in my opinion this is unusefull beacuse no devices in my LAN will contact unbound using TCP since the "normal" DNS is only in UDP.  

Thank you.

**Comment by ghost** [comment]
`do-tcp: no` should do, it does not appear to be impeding upstream DoT connectivity.

**Comment by iz8mbw** [comment]
In fact, `do-tcp` is not the good option for what I asked

**Comment by ghost** [comment]
Why not? It does what you want

* *unbound* does not listen on TCP ports
* DoT upstream still works

**Comment by iz8mbw** [comment]
DoT upstream does not work with `do-tcp: no`

**Comment by ghost** [comment]
Indeed, just did not ran a packet dump initially, just noticed that resolution still worked. But then a packet dump revealed that upstream traffic is switched to UDP
